messages/de/app.php

```diff
@@ -167,17 +167,17 @@
     'User' => 'Benutzer',
     'Users' => 'Benutzer',
     'View' => 'Ansehen', // view server button on Dashboard
-    'Want to be a partner?' => 'Du möchtest Partner werden?',
+    'Want to be a partner?' => 'Möchtest du Partner werden?',
     'Why don\'t you review your favorite <span class="discord-logo">Discord</span> server?' => 'Warum bewertest du nicht deinen Lieblings <span class="discord-logo">Discord</span> Server?',
     'Why don\'t you <a href="{reviewUrl}">review <strong>{serverName}</strong></a>?' => 'Warum bewertest du nicht <a href="{reviewUrl}"><strong>{serverName}</strong></a>?',
     'You have already muted {serverName}' => 'Du hast {serverName} bereits stummgeschaltet',
-    'You haven\'t added a server yet' => 'Du hast noch keinen Server hinzugefügt.',
-    'You must be logged in to mute a server' => 'Du musst eingeloggt sein, um einen Server hinzuzufügen.',
-    'You must be a member of the server to review it' => 'Du musst Mitglied des Servers sein, um ihn zu rezensieren',
-    'You need to be in the server for at least {duration} days to review' => 'Du musst für mindestens {duration} Tage im Server sein, um ihn zu rezensieren',
+    'You haven\'t added a server yet' => 'Du hast bisher noch keinen Server hinzugefügt.',
+    'You must be logged in to mute a server' => 'Du musst eingeloggt sein, um einen Server stummzuschalten.',
+    'You must be a member of the server to review it' => 'Du musst ein Mitglied des Servers sein, um ihn zu bewerten',
```

Comment:
I don't see why a grammatically correct sentence should be made unnecessarily longer. The change from rezensieren → bewerten is fine.